node/src/models/Mentor.ts

```diff
@@ -81,16 +80,24 @@ export interface MentorModel extends Model<Mentor> {
   ): Answer[];
 }
 
+// Return subjects in alphabetical order
 MentorSchema.statics.getSubjects = async function (
   m: string | Mentor
 ): Promise<Subject[]> {
   const mentor: Mentor =
     typeof m === 'string' ? await this.findOne({ _id: m }) : m;
-  return await SubjectModel.find({ _id: { $in: mentor.subjects } }, null, {
-    sort: { name: 1 },
-  });
+  return await SubjectModel.find(
```

Comment:
Better to explicitly handle the case that no mentor found for the passed in `mentor` argument, so the test would look like

```typescript
it("errors when mentor not found", async() => {
  ...
  expect(body).to.have.deep.nested.property(
    'errors[0].message',
    'mentor {badId} not found'
  )
})
```